ddtrace/tracer.py

```diff
@@ -658,8 +659,8 @@ def _on_span_finish(self, span):
         if self.log.isEnabledFor(logging.DEBUG):
             self.log.debug("finishing span %s (enabled:%s)", span.pprint(), self.enabled)
 
-    def _initialize_span_processors(self):
-        # type: () -> None
+    def _initialize_span_processors(self, appsec_enabled=asbool(get_env("appsec", "enabled", default=False))):
```

Comment:
:+1: this is ok for now and we'd move this to `configure()` once appsec is public

Reply:
yes! I was not sure how to use it in this round